api/v1beta1/disruption_spancontext.go

```diff
@@ -0,0 +1,58 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023 Datadog, Inc.
+package v1beta1
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/propagation"
+)
+
+const (
+	annotationSpanContextKey = "SpanContext"
```

Comment:
I would expect to have something scoped to chaos, like [here](https://github.com/DataDog/chaos-controller/blob/main/api/v1beta1/disruption_webhook.go#L52)

I'm wondering why aren't we not also scoping the user info to chaos 🤔 that's a bug IMO (another PR)